Cdnfsd_TransparentCDN_Popup_View_Success.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace W3TC;
+
+if ( !defined( 'W3TC' ) )
+	die();
+?>
+<form class="w3tc_cdn_transparentcdn_fsd_form">
+    <div class="metabox-holder">
+        <?php Util_Ui::postbox_header(
+	__( 'Succeeded', 'w3-total-cache' ) ); ?>
+
+        <div style="text-align: center">
+            <?php __('Site was successfully configured.', 'w3-total-cache') ?> <br />
+        </div>
+
+        <p class="submit">
+            <input type="button"
+                class="w3tc_cdn_stackpath2_fsd_done w3tc-button-save button-primary"
```

Comment:
Found another reference to "stackpath2". Please update all files in this pull request to reference "transparent", to avoid conflicts.